tests/phpunit/integration/Core/Authentication/Clients/OAuth_ClientTest.php

```diff
@@ -307,20 +307,19 @@ function ( Request $request ) {
 	}
 
 	public function test_using_proxy() {
+		$this->setExpectedDeprecated( OAuth_Client::class . '::using_proxy' );
 		$context = new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE );
+		$client  = new OAuth_Client( $context );
```

Comment:
This same test needs to be added to `CredentialsTest` (only without the deprecation expectation of course 😄 )
The test itself will need to be slightly different but keep in mind that whenever we're adding a new public method for a class that it will generally need a test.

As for the `fake_*` methods.. these should probably be pulled into a shared trait that both test cases can use.

Reply:
All done.

I've made an attempt at a new trait :)